easybuild/tools/modules.py

```diff
@@ -587,9 +587,15 @@ def modpath_extensions_for(self, mod_names):
 
         modpath_exts = {}
         for mod_name in mod_names:
-            modtxt = self.read_module_file(mod_name)
-            useregex = re.compile(r"^\s*module\s+use\s+(\S+)", re.M)
-            exts = useregex.findall(modtxt)
+            # Should be module show so as to catch conditional loads.
+            modtxt = self.show(mod_name)
+            useregex = re.compile(r"^\s*module\s+use\s+(\S+)|^\s*prepend_path\(\"MODULEPATH\",\s*\"(\S+)\"", re.M)
```

Comment:
Yes, although since we are creating the line, we might be able to get away with only double.